client/package.json

```
@@ -43,7 +43,11 @@
     "lint:js": "npm run lint:eslint -- . ",
     "lint:js:fix": "npm run lint:eslint -- . --fix",
     "pretest": "npm run test:clean",
-    "test:clean": "rimraf ./coverage"
+    "test:clean": "rimraf ./coverage",
```

```suggestion
    "test:clean": "rimraf ./reports/coverage",
```
Since, in jest config file, the coverage directory is pointing to '<rootDir>/reports/coverage'